src/main/kotlin/com/squareup/cash/hermit/UI.kt

```diff
@@ -31,4 +34,18 @@ object UI {
 
         Notifications.Bus.notify(notification, project)
     }
+
+    fun explainProjectIsNotTrusted(project: Project) {
+        val message = "Untrusted Project: Running in safe mode."
```

Comment:
Maybe `Untrusted project: Hermit disabled`? I think it might be confusing to people what `Running in safe mode` means